docs/budget-org.md

```diff
@@ -0,0 +1,78 @@
+# The Sovryn Budget System
+
+There is a short term need to create a budget and organization structure that minimizes the number of transactions going through the Sovryn Treasury. 
+
+## Two-tier structure
+What is proposed is a two-tier structure, with the Sovryn Treasury at Tier 1 and organization “groups” at Tier 2.
+
+![](../images/sovryn-budget-org.png)
+
+Each group would have its own multisig composed of at least 3 members, with a majority vote  (e.g. 2-of-3) required to enact payments from the group to end recipients. To make budgeting for each group easier, groups are to be broken into functional units for each major area of work e.g. Product, Adoption, Operations, etc.
+
+## Budgeting
+Once groups are formed, there are two steps to creating the budget:
+1. Decide the total budget per period  
+2. Allocate the budget to groups
+
+One way to decide the budget is in a top down way. Total available funds are assessed and a budget is derived from that. For example, total available funds are 300 BTC, we want this to last at least 3 years, and so we allocate 50 BTC to Year 1, 100 BTC to Year 2, and 150 BTC to Year 3, anticipating that spending will increase as time goes on and the project grows. Groups will then negotiate with each other for a share of the total budget.
+
+Another way to decide the total budget is in a bottom up way. Here, groups will create budget proposals per period and come together to consider the total cost of all of the proposals and whether any of the proposals should be adjusted up or down based on the total cost. This method may be biased to overspending, since groups may try to request as much money as they think they can get away with during a given period, but with discipline and honest negotiation may also be a more realistic reflection of spending needs since the total budget is decided directly by the groups who need and will be making use of budgeted funds.
+
+Once the total budget and individual group allocations have been determined using one of the above methods (or another method not explored here) then groups can make proposals to the Sovryn Treasury to receive the funds.
+
+## Distributing funds to groups
+
+#### Option 1. Manual payments
+
```

Comment:
Hi @shallb thank you for your comment!

So I think the idea of delivering funds based on milestones makes sense. The thing is we want to try to minimize the number of proposals that have to go through the Sovryn Treasury since this requires a significant amount of attention and work from SOV holders. I could see the community deciding that this overhead is worth it to ensure accountability and follow-through on proposed projects. What I suggest as an alternative is that we allocate budgets to these group multisigs and then the group multisigs can decide how to further allocate from there -- and that could include providing funding to projects based on the completion of certain milestones. Note that this entails putting some trust in the multisig groups to be responsible stewards of the funds -- a tradeoff that may or may not be worth it for the reduced overhead for SOV holders.

[long background story that you can skip over if you want]

My proposed budget system here is informed by my experience with the Aragon Flock and Aragon Nest grants programs in the [Aragon](https://aragon.org] community. I'll share a bit of the background and then relate this story back to how the Sovryn Budget System I am proposing took its shape (and I'm not saying this proposed system is perfect or in its final form, but is "good enough" for our needs at this stage).

In Flock, we had large funding proposals, generally 6 months to 1 year worth of funding for full teams (devs, operations, marketing all working together) with several deliverables and initiatives. You can see one of them [here](https://github.com/aragon/flock/blob/b325c23fdb54f586b5207f6f9e79491ca9f092ec/teams/Aragon%20One/2019.md). This funding was intended for teams that had a proven track record and could be trusted to deliver on what they said they would and (most importantly) not simply abscond with the funds. This worked well enough I think, the biggest challenge that arose was that setting a year-long roadmap as a pre-product market fit startup inevitably ends up with a lot of changes throughout the year and so we had to grapple with how to stay agile and willing to adapt to market demands while remaining true to what was promised to the tokenholders who approved the initial funding proposal. We ended up giving teams some discretion to make these changes as long as they consulted with the community ahead of time to get buy-in and make sure there were no strong objections.

In Nest, proposals were smaller and project based, generally with milestone-based payouts. You can see what a Nest proposal looks like [here](https://github.com/aragon/nest/pull/172). Most teams that ended up getting a Flock grant started with a Nest grant. We would provide an initial payment to help the team pay their bills through the first milestone but they wouldn't receive funding for the next milestone until the first or current milestone was completed as specified. Again, we gave some room for discretion if the roadmap needed to change for various reasons, as long as there was upfront communication and buy-in on both sides (the team and the grant manager).  

Both of these programs had their pluses and minuses. But beyond the particulars of each program, there was a meta-problem: how much funding should be allocated to both Flock and Nest grants in a given time period? The Aragon Association (the nonprofit overseeing the Aragon project) had a large treasury worth around $50m when these programs were operational. On one extreme, we could spend all $50m in a single year. On another extreme, we could try to stretch this funding 100 years by only spending $500,000 per year. Either extreme was unacceptable but there's a lot of possibility in the middle. The Aragon community ended up [approving a proposal](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-103.md) to impose a budget cap of 5% of the Aragon Association Treasury per quarter. This cap could only be changed by passing a proposal to change core governance process (a proposal that would have a much high threshold for approval than a normal spending proposal).

For various reasons, we never got to see the real-world effects of imposing a budget in this way. The Aragon Association ended up suspending the proposal process and re-taking centralized control of budgeting and spending. There was later some experimentation with [Conviction Funding](https://aragon.org/blog/introducing-the-conviction-funding-pilot), where proposals were approved by the community using the Conviction Voting mechanism, but this program also had a budget decided in a top-down way by the Aragon Association.  
[/end long background story]

This comment is now a big wall of text so I'll wrap it up. How does this relate to your comment about milestones and the overall budget system? The point I want to make is that even if we do milestone-based funding, this doesn't necessarily answer the question of how much funding should be allocated towards projects in a given time period to begin with. Maybe the answer is "as much as needed". I think that answer could be acceptable, but would point out that it risks running into the issue we had at Aragon which was overspending due to saying "yes" to everything that sounded good. At Aragon we ended up going with a credibly neutral mechanism (a hard cap on spending) that limited our ability to say "yes" beyond a certain point. Sovryn might want to consider the same. And then within the boundaries of that hard cap budget, teams/projects can ask for funding and receive it on a milestone basis. 

The question this raises for me is then who monitors progress and controls the release of funds upon the completion of a milestone? SOV holders, group multisigs as specified in my proposed budget system, someone else? Let me know what you think!